content/writing/moving-to-a-four-day-week.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Moving to a four-day week"
+date: 2021-10-02
+intro: It's the future!
+---
+
+About three months ago we decided to trial a four-day week, at no loss of pay, for all members. We reviewed it yesterday and it easily passed as a permanent change to our employment statuses. In this post, we'll outline how we did it, what we discovered in our trial and why the four-day week is an important step for any organisation.
+
+### Why move to a four-day week?
+
+The idea of moving to a four-day week has been gaining a lot of traction in the last few years, and the pandemic has accelerated this trend. Governments and businesses around the world have already run successful trials:
+
+> According to pilot studies, workers reported anywhere from a 25% to 40% increase in productivity, as well as an improved work/life balance, less need to take sick days, more time to spend with family and children, less money spent on childcare, and a more flexible working schedule which leads to better morale.
+
+— From [A four-day workweek is the future. Here’s why](https://www.theguardian.com/commentisfree/2021/aug/24/four-day-workweek-legislation) by Congressman Mark Takano
+
+But it's not just about the benefits experienced by individual workers. The [Four Day Week campaign](https://www.4dayweek.co.uk/why) have an expansive list of reasons for moving to a four-day week and how this can have a wider impact on our society, including:
+
+- A fairer society: more equal share of paid and unpaid work, including of caring roles traditionally ascribed to women
+- Strengthened communities: more time to build relationships, to care for children, the elderly and the disabled.
+- A reduced carbon footprint: research has shown that a four-day week could reduce the UK's carbon footprint by 127 million tonnes per year, equivalent to taking 27 million cars off the road (effectively the entire UK private car fleet).
+- More time to engage with politics, participate in local campaigns and shape society at a local, national and international level.
+
+### Our hesitations
+
+As a worker co-operative, it's admittedly fairly simple to initiate a change like this. One of us made an initial proposal, we clarified it and addressed critical concerns as a group and then documented this decision on Loomio. 
```

Comment:
Maybe for a nice detail add in a link to:
1. Sociocracy to say how we made the decision
2. Make it clear we did this over the internet, as it is interesting.
3. Link to Loomio.